aito/utils/data_frame_handler.py

```diff
@@ -37,25 +42,45 @@ def validate_in_out_format(self, in_format: str, out_format: str):
 
     @staticmethod
     def datetime_to_string(df: pd.DataFrame) -> pd.DataFrame:
-        # Aito hasn't support datetime data yet. Converting all datetime data to string
+        """Convert pandas datetime type to string
+
+        :param df: input pandas DataFrame
+        :type df: pd.DataFrame
+        :return: converted pandas DataFrame
+        :rtype: pd.DataFrame
+        """
         for col in df:
             if df[col].dtypes == 'datetime64[ns]':
                 df[col] = df[col].astype(str)
         return df
 
     @staticmethod
     def apply_functions_on_df(df: pd.DataFrame, functions: List[Callable]) -> pd.DataFrame:
-        """
-        Applying functions sequentially to a dataframe
-        :param df:
-        :param functions:
-        :return:
+        """Applying partial functions to a dataframe
+
+        :param df: input pandas DataFrame
+        :type df: pd.DataFrame
+        :param functions: list of partial functions that will be applied to the loaded pd.DataFrame
+        :type functions: List[Callable]
+        :return: output DataFrame
+        :rtype: pd.DataFrame
         """
         for f in functions:
             df = f(df)
         return df
 
-    def convert_df_from_aito_table_schema(self, df: pd.DataFrame, table_schema: Dict):
+    def convert_df_from_aito_table_schema(self, df: pd.DataFrame, table_schema: Dict) -> pd.DataFrame:
```

Comment:
```suggestion
    def convert_df_using_aito_table_schema(self, df: pd.DataFrame, table_schema: Dict) -> pd.DataFrame:
```
Based on the name of the function and description I'm not entirely sure what the function does. The function name is df FROM aito schema but the description is df TO aito schema which in my mind is a different feature what the function name implies.